routes/userRoutes.js

```diff
@@ -153,4 +153,46 @@ router.post("/login", async (req, res, next) => {
     }
   });
   
+// PUT /api/user/friend-request
+router.put("/friend-request", auth, async (req, res, next) => {
+ 
+  const { receiverId } = req.body;
+  const sender = await findUserById(req.user._id);
+  const receiver = await findUserById(receiverId);
+
+  if (!receiver) {
+    return res.status(404).json({ msg: "Empfänger nicht gefunden" });
+  }
+
+  const response = await handleFriendRequest(sender, receiver);
+  return res.status(response.status).json({ msg: response.msg });
+
+});
+
+// Put /api/user/accept-friend-request
+router.put("/accept-friend-request", auth, async (req, res, next) => {
+  const { userId } = req.body;
+  const receiver = req.user;
+  const sender = await findUserById(userId);
+
+  if (!receiver) {
+    return res.status(404).json({ msg: "User nicht gefunden" });
```

Comment:
```suggestion
    return res.status(404).json({ msg: "Benutzer nicht gefunden" });
```

Würds am besten einheitlich halten, also entweder immer "User" oder immer "Benutzer". Oder User wenn englisch genutzt wird und benutzer wenn deutsch

Reply:
kk